src/index.js

```diff
@@ -3,9 +3,6 @@ import sizeMe from 'react-sizeme';
 
 sizeMe.noPlaceholders = true;
 
-// Styles
-export styles from './styles.scss';
```

Comment:
Storybook relied on this for the styles to be included I believe. Styles are missing/invalid in the storybook deploy preview currently:

![image](https://user-images.githubusercontent.com/3360588/77930765-e8735e80-7270-11ea-846d-8cd3e2ce31f2.png)

I think to fix you can import the styles.scss entrypoint to `.storybook/story-styles.scss`. Styles in that file are applied to all stories.

Reply:
added to storybook styles